bigtable/tests/filters_integration_test.cc

```diff
@@ -133,6 +110,7 @@ TEST_F(FilterIntegrationTest, PassAll) {
   CreateCells(*table, expected);
 
   auto actual = ReadRows(*table, bigtable::Filter::PassAllFilter());
+  DeleteTable(table_name);
```

Comment:
An idea for the future: set a member variable in the fixture and get `TearDown()` to delete the tables, e.g., we could add:

```C++
std::vector<std::string> tables_to_be_deleted_;
```

Then `CreateTable()` could push the table name to it, and then:

```C++
void TearDown() {
  for (auto const& table_name : tables_to_be_deleted_) {
    DeleteTable(table_name);
  }
```